### ISSUE TITLE
XR 6.3.2, tailf import missing

### ISSUE THREAD

**Issue by rgaglian** [YangModels/yang#397]
Hi,

Found this issue:

yang/Cisco-IOS-XR-sysadmin-entity-mib.yang:90: error: prefix 'tailf' not found
yang/Cisco-IOS-XR-sysadmin-entity-mib.yang:130: error: prefix 'tailf' not found

Roque

**Comment by einarnn** [comment]
Yeah, we have a couple of errors in the IOS XR 6.3.2 and 6.4.1 model uploads. I’m trying to carve out the time to get them fixed. Part of the fix will include trapping the error below. I haven’t quite kept the CI checking scripts properly up to date.

Cheers,

Einar
